extensions/blocks/subscriptions/deprecated/v3/get-subscriptions-shortcode.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+import {
+	getColorClassName,
+	__experimentalGetGradientClass as getGradientClass,
+	getFontSizeClass,
+} from '@wordpress/block-editor';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import definedAttributes from './attributes';
+import {
+	DEFAULT_BORDER_RADIUS_VALUE,
+	DEFAULT_BORDER_WEIGHT_VALUE,
+	DEFAULT_PADDING_VALUE,
+	DEFAULT_SPACING_VALUE,
+	DEFAULT_FONTSIZE_VALUE,
+} from '../../constants';
+
+export default function getSubscriptionsShortcode( className, attributes, checkTextDefaults = null ) {
+	const {
+		subscribePlaceholder,
```

Comment:
The error I'm seeing seems to be related to subscribePlaceholder not existing. Possibly an issue with the deprecation script?

Reply:
Thanks for testing this. I've fixed up the deprecation scripts and it tests well for me now.